site-manage/pageorder/tool/src/webapp/content/css/PageList.css

```diff
@@ -69,6 +69,21 @@ li.sortable_hover
 	padding: 4px;
 	background-color: var(--sakai-background-color-3);
 }
+
+.grab_cursor
+{
+	cursor: grab;
+	cursor:-moz-grab;
+	cursor: -webkit-grab;
```

Comment:
Browser prefix properties should go before the non-prefixed standard properties:
```
cursor:-moz-grab;
cursor: -webkit-grab;
cursor: grab;
```

The -moz prefix hasn't been needed since 2013, so I don't think we need to include it: https://caniuse.com/?search=grabbing

```suggestion
	cursor: -webkit-grab;
	cursor: grab;
```

Reply:
Do we even need the vendor prefixes? Seems like it's had full support for a good while https://caniuse.com/css3-cursors-grab

Reply:
As I mentioned above, -moz has not been needed since 2013, so I think we can drop that, as I suggested. 

-webkit has not been needed since 2018, so that one I could go either way.

Reply:
Good to know, I'm removing it.